libraries/AP_Vehicle/AP_Vehicle.cpp

```diff
@@ -258,6 +268,9 @@ const AP_Scheduler::Task AP_Vehicle::scheduler_tasks[] = {
 #if HAL_EFI_ENABLED
     SCHED_TASK_CLASS(AP_EFI,       &vehicle.efi,            update,                   10, 200),
 #endif
+#if AP_HYGROMETER_ENABLED
+    SCHED_TASK_CLASS(AP_Hygrometer, &vehicle.hygrometer,      update,                   3,  100),
```

Comment:
This will currently be conflicting; you have to add a priority number in here now